roadmap/triggers.md

```diff
@@ -0,0 +1,24 @@
+# Tekton Triggers Roadmap
+
+This doc describes the roadmap for [Tekton Triggers](https://github.com/tektoncd/triggers).
+
+In 2019 we created a simple system for creating instances of Tekton resources, triggered
+by json payloads sent to HTTP endpoints (Tekton Triggers).
+
+In 2020 we would like to add missing features and then push for
+[Triggers to be Beta and later GA](#beta-and-ga):
+
+We are targetting improving the experience for both end users and operators:
```

Comment:
Optional integration with knative maybe ?

Reply:
Can you explain a bit more here? I'm still of the opinion that our best bet is to be as generic as possible

Reply:
Hence "optional" in my comment :stuck_out_tongue: An example would be : using a knative service to expose the EventListener instead of a standard K8S service. I should probably have written "optional integration with knative/serving" maybe :stuck_out_tongue_closed_eyes:

Reply:
I guess that could be under improved support for many EventListeners?

Reply:
Im unclear of the value in exposing the listener via knative service rather than corev1.Service, and am curious about the improvement it would provide. I do love me so knative serving/eventing and use this combo with triggers regularly but feel like im missing the background.

Reply:
@iancoffey not too much, just would like to be able to have a "scale-to-zero" service for event-listener

Reply:
Maybe I can include this as "scale-to-zero EventListner"?

Reply:
yeah works for me :wink: